### DISCUSSION TITLE
Display originating application

### DISCUSSION THREAD

**Discussion by jasonvarga** [spatie/ray#557] [Ideas]
If I use `ray()` from two different apps at the same time, there is no way to distinguish between the calls (other than hovering over the filename to see the full path).

Perhaps Ray could show the hostname in the output?

**Comment by freekmurze** [comment]
This is a good idea, I'll see how we can implement that in a good way.

A thing that I sometimes do when debugging multiple apps, is to add a specific color for each app for a logged item (it's a bit cumbersome, but it works)

**Reply by jasonvarga** [reply]
Aren't you supposed to be on vacation? 😄

**Reply by freekmurze** [reply]
Lol 👀 

While waiting on the plane back home I allow myself answers to issues 🙂
